nesc/whip6/platforms/boards/cc26xxbased/NonBlockingSCUARTPub.nc

```diff
@@ -0,0 +1,22 @@
+/*
+ * whip6: Warsaw High-performance IPv6.
+ *
+ * Copyright (c) 2017 University of Warsaw
+ * All rights reserved.
+ *
+ * This file is distributed under the terms in the attached LICENSE
+ * files.
+ */
+#ifndef SC_UART_BAUD_RATE
+#define SC_UART_BAUD_RATE 115200
```

Comment:
Supervisor uses 57600 by default, and it's pretty hard to change (kernel console, uboot). Could we use that speed here as the default too?